pkgs/by-name/at/atuin-desktop/package.nix

```
@@ -0,0 +1,131 @@
+{ lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  wrapGAppsHook4,
+  dbus,
+  zlib,
+  fontconfig,
+  openssl,
+  libappindicator,
+  webkitgtk_4_1,
+  gtk3,
+  pango,
+  cairo,
+  gdk-pixbuf,
+  libsoup_3,
+  glib,
+  glib-networking,
+  gcc,
+  pkg-config,
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "atuin-desktop";
+  version = "0.1.3";
+
+  src = fetchurl {
+    url = "https://github.com/atuinsh/desktop/releases/download/v${version}/Atuin_Desktop_${version}_amd64.deb";
+    sha256 = "sha256:6d30da5ee205d26ebaa062e5dd1c8f35702c194f38af21bb51fe07086c037e0f";
```

Can you use a SRI-hash? 
```suggestion
    hash = "sha256-...";
```